doc/nrf/releases_and_maturity/releases/release-notes-2.6.0.rst

```diff
@@ -845,7 +845,7 @@ Matter samples
 * Disabled the following features:
 
   * :ref:`ug_matter_configuring_read_client` in most Matter samples using the new :kconfig:option:`CONFIG_CHIP_ENABLE_READ_CLIENT` Kconfig option.
-  * WPA supplicant advanced features in all Matter samples using the :kconfig:option:`CONFIG_WPA_SUPP_ADVANCED_FEATURES` Kconfig option.
+  * WPA supplicant advanced features in all Matter samples using the :kconfig:option:`CONFIG_WIFI_NM_WPA_SUPPLICANT_ADVANCED_FEATURES` Kconfig option.
```

Comment:
I don't think we should rename old release notes. Normally I've just removed linking so doc build succeeds but tech writers can comment:
```suggestion
  * WPA supplicant advanced features in all Matter samples using the ``CONFIG_WPA_SUPP_ADVANCED_FEATURES`` Kconfig option.
```

Reply:
Ah, I just reverted this change, I will update to remove the linkage to Kconfig.

Reply:
Agree. Change it to ``CONFIG_WPA_SUPP_ADVANCED_FEATURES``.

Reply:
Already did, thanks.